.github/helm/affine/templates/ingress.yaml

```diff
@@ -74,4 +74,11 @@ spec:
                 name: affine-web
                 port:
                   number: {{ .Values.web.service.port }}
+          - path: /js/worker.(.+).js
```

Comment:
should we set the `cache-control` header for this path because the worker filename contains no hash?

Reply:
it includes hash now